solidus_jwt.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'jwt'
   s.add_dependency 'solidus_auth_devise'
-  s.add_dependency 'solidus_core', ['>= 2.0.0', '< 4']
```

Comment:
why `>= 3.2.0`?

Reply:
🤔 I guess for this one it doesn't matter, I will change it back